dogfooding/src/main/kotlin/io/getstream/video/android/ui/call/SettingsMenu.kt

```diff
@@ -54,6 +59,21 @@ internal fun SettingsMenu(
     val reactions =
         listOf(":fireworks:", ":hello:", ":raise-hand:", ":like:", ":hate:", ":smile:", ":heart:")
 
+    val screenSharePermissionResult = rememberLauncherForActivityResult(
+        contract = ActivityResultContracts.StartActivityForResult(),
+        onResult = {
+            if (it.resultCode == Activity.RESULT_OK && it.data != null) {
+                call.startScreenSharing(it.data!!)
+            }
+            onDismissed.invoke()
+        },
+    )
```

Comment:
I think we can make this simpler by building dedicated Compose APIs. Let me create one once we merge this PR.

Reply:
Nice. Yes - I was thinking if we maybe can help a bit more with the integration if we provide something in the SDK to handle this permission.